src/components/login/LoginModal.tsx

```
@@ -25,13 +30,10 @@ export default function LoginModal({ isOpen, onClose }: LoginModalProps) {
   const handleSubmit = (e: React.FormEvent) => {
     e.preventDefault()
 
-    // 로그인 로직 추후 구현
-    if (false) {
-      console.log("Login attempt with:", { email, password })
-    } else {
-      setErrorMessage("이메일과 비밀번호를 확인해주세요.")
-    }
-    
+    apiCall("/api/user/login", "POST", { email, password }).then(({ data }: { data: any }) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)